### ISSUE TITLE
NTR: Anti-THSD7A antibody present in blood (Lab: serology & immunological tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8785]
New term request Anti-THSD7A antibody present in blood (nadea3_190821161049): Thrombospondin type 1 domain containing 7A

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000187
- label: Anti-thrombospondin type I domain-containing 7A antibody positivity
- parents: [Autoimmune antibody positivity (HP:0030057)](https://hpo.jax.org/app/browse/term/HP:0030057)
- pmid: PMID:35564696
- def: The presence of autoantibodies (immunoglobulins) in the blood circulation that react against thrombospondin type I domain-containing 7A.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8785
- synonyms: Anti-THSD7A antibody present in blood(NONE)|Anti-THSD7A antibody(NONE)
